torch/csrc/tensor/python_tensor.h

```diff
@@ -27,9 +27,9 @@ TORCH_PYTHON_API void py_set_default_dtype(PyObject* dtype_obj);
 // TODO: This is nuts!  There is no reason to let the default tensor type id
 // change.  Probably only store ScalarType, as that's the only flex point
 // we support.
-TORCH_API c10::DispatchKey get_default_dispatch_key();
+TORCH_PYTHON_API c10::DispatchKey get_default_dispatch_key();
 TORCH_PYTHON_API at::Device get_default_device();
 
 // Gets the ScalarType for the default tensor type.
-at::ScalarType get_default_scalar_type();
+TORCH_PYTHON_API at::ScalarType get_default_scalar_type();
```

Comment:
Maybe we should test if this change affects torch_npu cc: @FFFrog

Reply:
I guess it will just bring some symbols back. Those symbols used to exist but got removed ~3 months ago.

Reply:
No effect, but adding TORCH_PYTHON_API for private functions is a bit weird to me :/

Reply:
You're right. So those are moved to `public` as discussed in https://github.com/pytorch/pytorch/issues/148208.